src/main/java/net/dv8tion/jda/internal/requests/restaction/ChannelActionImpl.java

```diff
@@ -195,6 +213,23 @@ public ChannelActionImpl<T> setSlowmode(int slowmode)
         return this;
     }
 
+    @Nonnull
+    @Override
+    public ChannelAction<T> setDefaultReaction(@Nullable Emoji emoji)
+    {
+        this.defaultReactionEmoji = emoji;
+        return this;
+    }
+
+    @NotNull
+    @Override
+    public ChannelAction<T> setAvailableTags(@NotNull List<? extends BaseForumTag> tags)
+    {
+        Checks.noneNull(tags, "Tags");
+        this.availableTags = new ArrayList<>(tags);
+        return this;
+    }
```

Comment:
We should probably throw if you try to set these on a non-forum channel like we do for audio in setBitrate